lib/signaling/v2/room.js

```diff
@@ -765,8 +765,8 @@ function periodicallyPublishStats(roomV2, transport, intervalMs) {
       stats.forEach((response, id) => {
         // NOTE(mmalavalli): A StatsReport is used to publish a "stats-report"
         // event instead of using StandardizedStatsResponse directly because
-        // StatsReport will add nulls to properties that do not exist.
-        const report = new StatsReport(id, response);
+        // StatsReport will add zeros to properties that do not exist.
+        const report = new StatsReport(id, response, true /* prepareForInsights */);
```

Comment:
@makarandp0 I was thinking we could leverage [replaceNullWithDefaults](https://github.com/twilio/twilio-video.js/blob/master/lib/signaling/v2/room.js#L825) in RoomV2? Then we won't need to change `StatsReport` which is a public facing class.

Reply:
@manjeshbhargav - I looked into reusing something like `replaceNullWithDefaults` but things get more complicated with it. 

I think  `replaceNullsWithDefaults` was necessary for `activeIceCandidatePair` because it operates on response from getStats, and does not do any property renaming. Where as for stats-report we have a good implementation of StatsReport does that renaming and folding of properties. For example: 
`response.frameRateInput` becomes `report.captureFrameRate`

I think it's better to let  statsReport manage defaults instead of overwriting them outside it.